woocommerce/woocommerce-android
# [Woo POS] Custom card component to remove simulation of the light source from the top right

Closes: #12492


### Description

The PR implements custom card components that draw a shadow for elevation without taking into account light source.

Also the PR adds shadow to the bottom of the cart list

Discussion - p1725009492618169-slack-C070SJRA8DP

### Testing information

* Use the POS and notice that the shadows that illustrates elevation on the cards are drawn below the elements.
* Notice that there is a shadow appears when cart is scrollable but not at the bottom
* Put extra attention on potential performance degradation - scrolling of the lists

### The tests that have been performed

I looked into what I mentioned into the testing information

### Images/gif


https://github.com/user-attachments/assets/d65f00c8-5cf1-4da0-92ce-29cf2f5c18bc



## Reviewer (or Author, in the case of optional code reviews):

Please make sure these conditions are met before approving the PR, or request changes if the PR needs improvement:
